.github/workflows/iotlab-ci.yml

```
@@ -0,0 +1,90 @@
+name: iotlab-ci
+
+# Run 'compile_and_test_for_board.py' on IoT-LAB boards.
+#
+# This workflow will launch all tests on a list of IoT-LAB boards.
+#
+# This workflow is only run periodically on master with a cron and on new
+# pushed release candidates tags.
+
+on:
+  # Schedule weekly runs Sunday at 00:00 on master
+  schedule:
+    - cron: '00 0 * * 0'
+  push:
+    # TODO: remove master branch, only used here as demonstrator
+    branches:
+      - master
+    # Run on all new release candidates tags
+    tags:
+      - '*-RC*'
+
+jobs:
+  # Runs all tests on IoT-LAB boards
+  compile_and_test_for_board:
+    strategy:
+      max-parallel: 10
+      fail-fast: false
+      matrix:
+        boards:
+          - '{"riot": "iotlab-m3", "iotlab": "m3:at86rf231", "site": "saclay"}'
```

Maybe also do something like this?

```suggestion
          - riot: iotlab-m3
            iotlab:
              archi: "m3:at86rf231"
              site: saclay
```